### PR TITLE
[Storage][ChangeLog]STG77 Changelog

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by tasherif-msft** [comment]
So it looks like we're also gonna be doing a patch release for blobs due to https://github.com/Azure/azure-sdk-for-python/pull/18690

File: sdk/storage/azure-storage-file-datalake/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.4.0b1 (2021-05-12)
+**New features**
+- Added support `set_service_properties()`,`get_service_properties()` on `DataLakeServiceClient`
+- Added support for `list_deleted_paths()` on `FileSystemClient`
+
```
Comment (tasherif-msft) [inline sdk/storage/azure-storage-file-datalake/CHANGELOG.md:7]: lets merge https://github.com/Azure/azure-sdk-for-python/pull/18559 and add it here as well!

Comment (xiafu-msft) [reply sdk/storage/azure-storage-file-datalake/CHANGELOG.md:7]: Added

**Comment by xiafu-msft** [comment]
> So it looks like we're also gonna be doing a patch release for blobs due to #18690 so let's do its CHANGELOG as well!

During Thursday standup I asked Ashish if the service change is considered as breaking change he said no, so I didn’t address this comment.
I just confirmed with Sean it’s wasn’t mentioned  in .Net changelog either, so let’s just leave it like this.

**Review by amnguye — APPROVED** [review]

**Review by gapra-msft — APPROVED** [review]

**Pull request merged**